.github/workflows/test_ci_scripts.yml

```
@@ -2,7 +2,13 @@
 name: "CI/CD Infrastructure Tests"
 
 on:  # yamllint disable-line rule:truthy
-  workflow_call:   # yamllint disable rule:empty-values
+  workflow_call:
+    inputs:
+      godot_version:
+        description: "The Godot version string passed from the calling pipeline"
+        type: string
+        default: "4.6.3-stable"
```

actionlint (1.7.12)

[error] input "godot_version" of workflow_call event has the default value "4.6.3-stable", but it is also required. if an input is marked as required, its default value will never be used

(events)